pkgs/applications/virtualization/qemu/default.nix

```
@@ -222,7 +237,8 @@ stdenv.mkDerivation rec {
     if [ -x $out/bin/qemu-system-${stdenv.hostPlatform.qemuArch} ]; then
       makeWrapper $out/bin/qemu-system-${stdenv.hostPlatform.qemuArch} \
                   $out/bin/qemu-kvm \
-                  --add-flags "\$([ -e /dev/kvm ] && echo -enable-kvm)"
+                  --run ${emit-kvm-warnings}/bin/emit-kvm-warnings \
+                  --add-flags "\$([ -r /dev/kvm ] && [ -w /dev/kvm ] && echo -enable-kvm)"
```

Nitpick: no need for two invocations of `[` here.

```suggestion
                  --add-flags "\$([ -r /dev/kvm -a -w /dev/kvm ] && echo -enable-kvm)"
```

---

Done in d033935ab6fd850160e31a1db822486f39650210